docs/docs/deploy/clients.md

```diff
@@ -3,6 +3,9 @@ layout: default
 title: Install Clients
 nav_order: 5
 parent: Deploy
+description: >
+  Firezone is compatible and uses the official WireGuard clients available
```

Comment:
Suggestion: remove `and uses` and clarify to `Firezone is compatible with the official WireGuard clients`

Reply:
fixed